packages/wpcom-checkout/src/checkout-line-items.tsx

```diff
@@ -1137,6 +1169,7 @@ function CheckoutLineItem( {
 						<DomainDiscountCallout product={ product } />
 						<CouponDiscountCallout product={ product } />
 						<IntroductoryOfferCallout product={ product } />
+						<JetpackAkismetSaleCouponCallout product={ product } />
```

Comment:
I haven't looked at all the logic, but is there a chance that this might print the same text as `IntroductoryOfferCallout`?

Reply:
Yes, it will be the same text, but this is checking for a sale coupon when an intro offer is absent.